### PR TITLE
[Fix] fix ncnn

### PR DESCRIPTION
**Pull request by lzhangzz**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

ncnn was not taken into consideration in #106, this PR tries to bring it back.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Review by hanrui1sensetime — APPROVED** [review]
LGTM.

File: csrc/backend_ops/ncnn/pyncnn_ext/CMakeLists.txt [hunk 1]
```diff
@@ -10,6 +10,8 @@ endif ()
 pybind11_add_module(ncnn_ext ncnn_ext.cpp)
 
 target_link_libraries(ncnn_ext PUBLIC mmdeploy_ncnn_ops ncnn)
-set_target_properties(
-        ncnn_ext PROPERTIES LIBRARY_OUTPUT_DIRECTORY
-        ${CMAKE_SOURCE_DIR}/mmdeploy/backend/ncnn)
+set(_NCNN_EXT_DIR ${CMAKE_SOURCE_DIR}/mmdeploy/backend/ncnn)
+set_target_properties(ncnn_ext PROPERTIES
+        LIBRARY_OUTPUT_DIRECTORY ${_NCNN_EXT_DIR}
+        LIBRARY_OUTPUT_DIRECTORY_DEBUG ${_NCNN_EXT_DIR}
```
Comment (lvhan028) [inline csrc/backend_ops/ncnn/pyncnn_ext/CMakeLists.txt:16]: Are both `LIBRARY_OUTPUT_DIRECTORY_DEBUG` and `LIBRARY_OUTPUT_DIRECTORY_RELEASE` properties necessary?

Comment (lzhangzz) [reply csrc/backend_ops/ncnn/pyncnn_ext/CMakeLists.txt:16]: On MSVC, the path to the target will contain "Debug" or "Release" otherwise.

**Review by lvhan028 — APPROVED** [review]
LGTM

**Pull request merged**